tables/hbo.tbl

```diff
@@ -0,0 +1,208 @@
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+
+#The first Hebrew Braille table hosted on Lib Louis was developed by the Library for the Blind and the Ministry of Education in Israel. That  table is based on the IHBC which was developed in the mid-1930’s. Please see World Braille usage 3rd edition, p. 74. 
+#However, Classical or Biblical Hebrew, which is used in the Hebrew Bible and other liturgical literature, contains cantillation marks that are not supported by the Hebrew table mentioned above. This table seeks to provide access to these accents. This table is   also based upon the tables mentioned above.  This table only departs from it where the accents are concerned. With this table, the user will have access to the accents and Masorah in BHS and BHS Quinta. For more information about the IHBC and How it was developed:
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#-maintainer: Paul Geoghegan <contact@envisionly.tech>
+#-maintainer: Matityhau Yeshurun <yeatersink@gmail.com>
+
+# always op code characters
+always ִי 35  # hiriq-yod
+always ֵי 3456  # tsere-yod
+always ּו 136  # shuruq
+always ּוְ 136-3  # shuruq sheva
+always בְּ 12-3  # bet with dagesh sheva
+always בּ 12  # bet with dagesh
+always דּ 5-145  # dalet with dagesh
+always דְּ 5-145-3  # dalet with dagesh sheva
+always וֹ 246  # holam-vav
+always וּ 136  # shuruq
+always יּ 5-245  # Yod with dagesh
+always יְּ 5-245-3  # Yod with dagesh sheva
+always כּ 13  # kaf with dagesh
+always כְּ 13-3  # kaf with dagesh sheva
+always לּ 5-123  # lamed with dagesh
+always לְּ 5-123-3  # lamed with dagesh sheva
+always מּ 5-134  # mem with dagesh
+always מְּ 5-134-3  # mem with dagesh sheva
+always נּ 5-1345  # nun with dagesh
+always נְּ 5-1345-3  # nun with  dagesh sheva
+always פּ 5-1234  # pe with dagesh
+always פְּ 5-1234-3  # pe with dagesh sheva
+always קּ 5-12345  # qof with dagesh
+always קְּ 5-12345-3  # qof with dagesh sheva
+always שְׁ 146-3  #  Shin with dot and sheva
+always שְׂ 156-3  #  Sin sheva
+always שׁ 146  # shin with dot
+always שׂ 156  # sin
+always ת 1456  # tav
+always תּ 2345  # tav with dagesh
```

Comment:
Dots are incorrect here. Tav with dagesh is 1256. The dots here 2345 are for a tet.Did you mean to have tav with dagesh in here twice?

Reply:
Yes, I'd like to see that article. You can just send it to my email if you like.
About the vav, I just think it may be better to label it vav with dagesh sheva.
I am doing a lot of work now with Mishnaic Hebrew and Talmud. The Hebrew is certainly different and interesting.